Makefile

```diff
@@ -3,7 +3,7 @@
 # Just for making development convenient
 #
 
-all: build-web build-backend
+all: build-backend
```

Comment:
The web build is needed ( https://github.com/mailtutan/mailtutan/pull/63 )

the web has to be built and the assets copied to lib.